### PR TITLE
CATTY-58: The function POW doesn't exist in Pocket Code

### PR DESCRIPTION
**Pull request by afibian**
Change the .tag attribute of the power function to 'POWER' to be inline with Catroid
Also change class name: PowFunction -> PowerFunction
and filename: PowFunction.swift -> PowerFunction.swift

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catty/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *#ios-general* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: src/CattyTests/Formula/FormulaParserTest.m [hunk 1]
```diff
@@ -361,7 +361,8 @@ - (void) testOperatorLeftBinding
 - (void) testOperatorChain
 {
     NSMutableArray *internTokenList = [[NSMutableArray alloc] init];
-    [internTokenList addObject:[[InternToken alloc] initWithType:TOKEN_TYPE_FUNCTION_NAME AndValue:@"POW"]]; // TODO use Function property
+//    [internTokenList addObject:[[InternToken alloc] initWithType:TOKEN_TYPE_FUNCTION_NAME AndValue:@"POW"]]; // TODO use Function property
```
Comment (aeluna) [inline src/CattyTests/Formula/FormulaParserTest.m]: Please remove comment.

Comment (afibian) [reply src/CattyTests/Formula/FormulaParserTest.m]: sorry, on it

Comment (afibian) [reply src/CattyTests/Formula/FormulaParserTest.m]: sorry, comment is deleted in d2a661d

**Comment by afibian** [comment]
deleted commented out code

**Pull request merged**